compiler/rustc_mir_transform/src/coverage/query.rs

```diff
@@ -109,6 +109,7 @@ fn coverage_ids_info<'tcx>(
         // to any particular point in the control-flow graph.
         // (Keep this in sync with the injection of `ExpressionUsed`
         // statements in the `InstrumentCoverage` MIR pass.)
+        // FIXME: sync me
```

Comment:
This one is a semantic sync rather than a data sync, and violating it would cause a bunch of tests to break, so it's probably not worth the hassle of adding sync-check tags here.